core/tests/protractor.conf.js

```diff
@@ -88,6 +88,10 @@ exports.config = {
       'protractor/learnerDashboard.js',
     ],
 
+    collectionEditor: [
+      'protractor/nodeCount.js'
```

Comment:
Nope -- bundle this test into protractor/collections.js instead. At the moment this won't run on Travis anyway because travis.yml isn't modified, and also nodeCount.js seems to be named at the wrong level of granularity (look at the other test file names).

Reply:
Ok, then can I remove the visits collection player test in collections.js as in these tests, it visits the player anyways?